students/mgummel/lesson07/html_render.py

```diff
@@ -0,0 +1,272 @@
+#!/usr/bin/env python3
+
+"""
+A class-based system for rendering html.
+"""
+
+
+class Element(object):
+    """
+    An Element class used to render different html class types. 
+    """
+    tag = "html"
+    attrs = ""
```

Comment:
this shouldn't be a class variable.

Think of class variable as something that never changes with each instance of a class, like it will always be "html" tag. 

However, attrs can be different with each instance.